### PR TITLE
Fix typo of orgWhitelist value

### PR DESCRIPTION
**Pull request by jbuettnerbild**
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md

### CODE REVIEW

**Comment by lkysow** [comment]
Sorry I'm just seeing this now. For some reason I didn't get auto-assigned. Thanks for the PR! You'll need to rebase off of master and then you also have to bump the Chart version for the PR to be mergeable.

**Comment by jbuettnerbild** [comment]
maybe i did't merge correctly.

File: stable/atlantis/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "v0.7.1"
 description: A Helm chart for Atlantis https://www.runatlantis.io
 name: atlantis
-version: 3.0.0
```
Comment (davidkarlsen) [inline stable/atlantis/Chart.yaml:5]: This should not be removed but rather bumped

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

File: stable/atlantis/Chart.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
...
+version: 3.0.1
```
Comment (lkysow) [inline stable/atlantis/Chart.yaml]:
```suggestion
version: 3.1.1
```

Another PR was merged today, sorry for the pain.

File: stable/atlantis/README.md [hunk 3]
```diff
@@ -24,6 +24,7 @@ The following options are supported.  See [values.yaml](values.yaml) for more de
 
 | Parameter                                   | Description                                                                                                                                                                                                                                                                                               | Default |
 |---------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|
+| `allowRepoConfig`                           | Whether to allow the use of [atlantis.yaml files](https://www.runatlantis.io/docs/atlantis-yaml-reference.html).                                                                                                                                                                                          | `false` |
```
Comment (lkysow) [inline stable/atlantis/README.md:27]: So you're adding all of these back now which were removed by https://github.com/helm/charts/pull/13253. Please rebase and ensure this changeset only has your changes.

**Review by lkysow — CHANGES_REQUESTED** [review]

**Pull request closed without merging**